_topic_map.yml

```diff
@@ -409,6 +409,17 @@ Topics:
   - Name: Configuring kube-proxy
     File: configuring-kube-proxy
     Distros: openshift-enterprise,openshift-origin
+- Name: OVN-Kubernetes network provider
+  Dir: ovn-kubernetes-network-provider
+  Topics:
+  - Name: Learn about the OVN-Kubernetes network provider
+    File: about-ovn-kubernetes
+  - Name: Enabling multicast for a project
+    File: enabling-multicast
+    Distros: openshift-origin,openshift-enterprise
+  - Name: Disabling multicast for a project
```

Comment:
Is there a reason that this layout isn't the same as OpenShift SDN? In there you have a "Using" assembly that has both. Would probably be good to be consistent whichever way you choose.

Reply:
@bergerhoffer, agreed. My focus for this PR had been changes for ovn-kube. I'll create a separate PR to apply that change.